libs/admin/src/lib/components/admin-page/admin-page.component.ts

```diff
@@ -19,6 +19,7 @@ export class AdminPageComponent implements OnInit, OnDestroy {
   @ViewChild('templateDrawerPortalContent') templateDrawerPortalContent!: TemplateRef<unknown>;
 
   public workgroup?: Workgroup = undefined;
+  public user?: User = undefined;
```

Comment:
Not sure about coding conventions - but I'd remove the explicit `undefined` definition since it is implied by using `user?`.